text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,222 @@
+- Feature Name: `target_tier_policy`
+- Start Date: 2019-09-20
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+We should have an official, objective policy for adding new (tier 3) targets,
+and for raising targets to tier 2 (with rustup builds) or even tier 1.
+
+# Motivation
+[motivation]: #motivation
+
+Rust developers regularly implement new targets in the Rust compiler, and
+reviewers of pull requests for such new targets would like a clear, consistent
+policy to cite for accepting or rejecting such targets. Currently, individual
+reviewers do not know what overall policy to apply, and whether to apply solely
+their own judgment or defer to a Rust governance team.
+
+Rust developers regularly ask how they can raise an existing target to tier 2
+(and in particular how they can make it available via rustup), and occasionally
+ask what it would take to add a new tier 1 target. The Rust project has no
+clear policy for target tiers. People not only don't know, they don't know who
+to ask or where to start.
+
+(See https://forge.rust-lang.org/platform-support.html for more information
+about targets and tiers.)
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+At a high level, the three tiers break down as follows:
+
+- Tier 3 targets provide no guarantees of support.
+- Tier 2 targets will always build, but may not pass tests.
+- Tier 1 targets will always build and pass tests.
+
+Adding a new tier 3 target imposes minimal requirements; we focus primarily on
+avoiding disruption to other ongoing Rust development.
+
+Tier 2 and tier 1 targets place work on the Rust community as a whole, to avoid
+breaking the target. Thus, these tiers require commensurate efforts from the
+maintainers of the target, to demonstrate value and to minimize any disruptions
+to ongoing Rust development.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Requirements for each tier:
+
+## Tier 3 target policy
+
+At this tier, the Rust project provides no official support for a target, so we
+place minimal requirements on the introduction of targets.
+
+- No central decision is required to add a new tier 3 target. Reviewers may
+  always use their own best judgment regarding the quality of work, and the
+  suitability of a target for the Rust project.
+- If a reviewer wishes to consult a broader team for additional guidance, they
+  should contact the compiler team.
+- If a proposed target or target-specific patch substantially changes code
+  shared with other targets (not just target-specific code), the reviewer
+  should consult the compiler team.
+- If the proposer of a target wishes to appeal the rejection of a target, they
```

Comment:
These three bullet points could be sub-bullets of a single "Contact the compiler team when ... " top-level bullet.

Reply:
I moved approving teams out of the bullet lists of requirements entirely. That should help ensure that the bullet lists contain only requirements, so that they can be used as checklists, and quoted for point-by-point explanations within target proposals.